### ISSUE TITLE
Inconsistent markdown processing between github and wiki

### ISSUE THREAD

**Issue by LukeEff** [risk-of-thunder/R2Wiki#22]
I noticed in a recent PR I merged the markdown has not formatted as I expected in the wiki.Particularly, Most of the indented lists were not recognized by the live website.

Here's an example of  the discrepancy. 
![correct](https://user-images.githubusercontent.com/55714366/228039496-da74f46d-9c7e-463b-bca8-e39850a9db19.PNG)
![screen](https://user-images.githubusercontent.com/55714366/228039502-5aabd78f-26f6-4fd6-aaa8-ea79b2d8a8d4.PNG)

And here's the code: https://github.com/risk-of-thunder/R2Wiki/blob/master/docs/Mod-Creation/C%23-Programming/Debugging-Your-Mods.md?plain=1#L165-L189

If this is working as intended, is there a syntax issue (such as improper spacing) that results in it not processing properly?
Also, is there a markdown processing plugin that can be installed that better reflects the "live website"?

**Comment by xiaoxiao921** [comment]
Yeah I had to fix it manually, this is most likely due to mkdoc having a different processor than the one used by mkdocs I'm afraid

**Comment by LukeEff** [comment]
You're right, I'm looking through the wiki right now.

[mkdocs claims to use the Python Markdown library](https://www.mkdocs.org/user-guide/configuration/#formatting-options) for translating the Markdown to html.

I have successfully replicated the issue by using their interpreter via a script i threw together:
```python3
import markdown

with open("docs/Mod-Creation/C#-Programming/Debugging-Your-Mods.md", "r", encoding="utf-8") as input_file:
    text = input_file.read()
    html = markdown.markdown(text)
    print(html)
```

**Comment by LukeEff** [comment]
https://github.com/Python-Markdown/markdown/issues/3#issuecomment-63399983

Must be 4 spaces for indentation to be registered for python-markdown, while Github's website uses an internal one.
